src/components/views/TableCellContextMenu.tsx

```diff
@@ -48,13 +68,23 @@ export const ContextMenu: React.FC<{ view: EditorView; close: () => void }> = ({
     undefined
   )
 
+  const { rows } = useMemo(() => {
```

Comment:
Do we need the `useMemo` here? The entire component will be re-rendered when the user clicks.

Reply:
yes that makes sense, removed